stdlib/public/Platform/Platform.swift

```diff
@@ -176,30 +176,30 @@ public func openat(
   _ oflag: Int32,
   _ mode: mode_t
 ) -> Int32 {
-  return _stdlib_openat(fd, path, oflag, mode)
+  return _swift_swift_stdlib_openat(fd, path, oflag, mode)
 }
 
 public func fcntl(
   _ fd: Int32,
   _ cmd: Int32
 ) -> Int32 {
-  return _stdlib_fcntl(fd, cmd, 0)
+  return _swift_stdlib_fcntl(fd, cmd, 0)
 }
 
 public func fcntl(
   _ fd: Int32,
   _ cmd: Int32,
   _ value: Int32
 ) -> Int32 {
-  return _stdlib_fcntl(fd, cmd, value)
+  return _swift_stdlib_fcntl(fd, cmd, value)
 }
 
 public func fcntl(
   _ fd: Int32,
   _ cmd: Int32,
   _ ptr: UnsafeMutableRawPointer
 ) -> Int32 {
-  return _stdlib_fcntlPtr(fd, cmd, ptr)
+  return _swift_swift_stdlib_fcntlPtr(fd, cmd, ptr)
```

Comment:
This and a few other functions use the prefix `_swift_swift_stdlib_`. They should be `_swift_stdlib_`.

Reply:
What the heck? The perils of trying to get something for people to look at before closing time on Friday, I guess.